xbmc/guilib/guiinfo/VideoGUIInfo.cpp

```diff
@@ -437,6 +437,16 @@ bool CVideoGUIInfo::GetLabel(std::string& value, const CFileItem *item, int cont
       case LISTITEM_VIDEO_ASPECT:
         value = CStreamDetails::VideoAspectToAspectDescription(tag->m_streamDetails.GetVideoAspect());
         return true;
+      case LISTITEM_VIDEO_WIDTH:
+        value = std::to_string(tag->m_streamDetails.GetVideoWidth());
+        if (value == "0")
+          value = "";
+        return true;
+      case LISTITEM_VIDEO_HEIGHT:
+        value = std::to_string(tag->m_streamDetails.GetVideoHeight());
+        if (value == "0")
+          value = "";
```

Comment:
No need to first convert the zero to a string, then to reset the string.

```c++
if (tag->m_streamDetails.GetVideoHeight() > 0)
  value = std::to_string(tag->m_streamDetails.GetVideoHeight());
return true;
```

Reply:
I don't know enough about compiler optimisation to say what would be most efficient. What I've proposed works.  Can we just get it in?  Or feel free to edit as you think fit.

Reply:
> What I've proposed works. 

 No doubt, but we can do better. ;-)
 
 > Can we just get it in?

 Well, no.  Just copy the code snippet I posted (and adapt it for width). It should work as it is.

Reply:
> No doubt, but we can do better

Depends on your definition of better.....